src/kdbindings/connection_handle.h

```diff
@@ -206,4 +208,99 @@ class ConnectionHandle
     }
 };
 
+/**
```

Comment:
Personally, I wouldn't mix these changes in a single commit as they are rather distinct and don't logically belong together. But It's not a showstopper of course.

Reply:
Right, it's not ideal semantically.
Just treated this as a general fixup after merging the deferred connections, where the ScopedConnection slipped through :see_no_evil: